src/sc_kpm/sc_agent.py

```diff
@@ -14,40 +15,37 @@
 from sc_client.models import ScAddr, ScEvent, ScEventParams
 
 from sc_kpm.identifiers import QuestionStatus
-from sc_kpm.logging import get_kpm_logger
 from sc_kpm.sc_keynodes import Idtf, ScKeynodes
 from sc_kpm.sc_result import ScResult
 from sc_kpm.utils.action_utils import check_action_class
 
-_logger = get_kpm_logger()
-
 
 class ScAgentAbstract(ABC):
     def __init__(self, event_element: ScAddr, event_type: ScEventType) -> None:
         self._event_element = event_element
         self._event_type = event_type
         self._event: Optional[ScEvent] = None
+        self._logger = getLogger(f"{self.__module__}.{self.__class__.__name__}")
 
     @abstractmethod
     def __repr__(self) -> str:
         pass
 
     def _register(self) -> None:
         if self._event is not None:
-            _logger.warning("%s is almost registered", self.__class__.__name__)
+            self._logger.warning("This ScAgent is almost registered")
             return
         event_params = ScEventParams(self._event_element, self._event_type, self._callback)
         self._event = client.events_create(event_params)[0]
-        _logger.info("ScEvent of %s was registered, %s", self.__class__.__name__, repr(self._event_type))
+        self._logger.info("ScEvent was registered: %s - %s", repr(self._event_element), repr(self._event_type))
```

Comment:
```suggestion
        self._logger.info("Agent was registered  with ScEvent: %s - %s", repr(self._event_element), repr(self._event_type))
```

Reply:
I'll change to:
`Registered with ScEvent`
`Unregistered ScEvent`